dev_requirements/release-requirements.txt

```diff
@@ -0,0 +1,2 @@
+pypi-parker
+setuptools
```

Comment:
i think this is missing a couple of other deps
https://github.com/aws/aws-encryption-sdk-cli/commit/b32b4b25429d891ea148be6f1d99e10961f055a2#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R296

i think we need wheel and twine